src/frontend/src/handler/query.rs

```diff
@@ -154,15 +150,23 @@ pub async fn handle_query(
 
     // update some metrics
     if query_mode == QueryMode::Local {
-        latency_local_execution_timer.observe_duration();
+        fn duration_to_seconds(d: Duration) -> f64 {
```

Comment:
Why not use
https://doc.rust-lang.org/stable/std/time/struct.Duration.html#method.as_secs_f64 ?